Vester/Tests/DSCluster/DSC-IOLatency.Vester.ps1

```diff
@@ -0,0 +1,26 @@
+# Test file for the Vester module - https://github.com/WahlNetwork/Vester
+# Called via Invoke-Pester VesterTemplate.Tests.ps1
+
+# Test title, e.g. 'DNS Servers'
+$Title = 'DatastoreCluster - IOLatencyThresholdMillisecond'
+
+# Test description: How New-VesterConfig explains this value to the user
+$Description = 'Get the IOLatencyThresholdMillisecond setting of a datastorecluster'
```

Comment:
Since Vester gets and sets, we may want to avoid the "Get" language here, as it's possible end users would be setting the value with -Remediate